lib/reporters/junit.js

```diff
@@ -108,13 +108,21 @@ exports.run = function (files, opts, callback) {
                 errorCount: 0,
                 failureCount: 0,
                 tests: 0,
-                testcases: [],
-                name: name
+                testcases: {},
```

Comment:
@hugosenari sorry this took so long. Why converting from array to dictionary here?